src/corppa/poetry_detection/annotation/recipe.py

```diff
@@ -0,0 +1,172 @@
+"""
+This module provides custom recipes for Prodigy annotation. They were
+created with page-level text annotation in mind, and support annotating
+text with a reference image displayed beside the text (`annotate_page_text`),
+or annotating both text and image side by side (`annotate_text_and_image`).
+
+Referenced images must be served out independently for display.
+
+Example use:
+```
+prodigy annotate_page_text poetry_spans poetry_pages.jsonl --label POETRY,PROSODY -F ../corppa/poetry_detection/annotation/recipe.py --image-prefix http://localhost:8000/
+prodigy annotate_text_and_image poetry_text_image poetry_pages.jsonl --label POETRY,PROSODY -F ../corppa/poetry_detection/annotation/recipe.py --image-prefix http://localhost:8000/
```

Comment:
Consider removing the second label for these examples, since these tasks will only have one in the main use case